src/styles/blocks/card.scss

```diff
@@ -0,0 +1,120 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  transition: $transition-speed;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-text-color;
+    margin-top: 40px;
+  }
+
+  &:hover &__title {
+    color: $title-hover-color;
+  }
```

Comment:
The nesting of '&:hover &__title' is incorrect. The '&:hover' should be outside the '&__title' block to correctly apply the hover effect to the title when the card is hovered. Consider restructuring this to '.card:hover .card__title'.